runtime/CSharp/Antlr4.Runtime/Antlr4.Runtime.portable-net45.project.json

```diff
@@ -0,0 +1,11 @@
+{
+  "dependencies": {
+    "NETStandard.Library": "1.6.0",
```

Comment:
Obviously I don't doubt the correctness of this but just curious to know how a reference to NETStandard.Library 1.6 relates to NETStandard 1.6 and the netstandard1.1 framework which this supports.